lmnet/configs/core/segmentation/lm_bisenet_quantize_camvid.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 The Blueoil Authors. All Rights Reserved.
```

Comment:
Not certain about this, but should this be `Copyright 2019` or not?

Reply:
thanks 👍

Reply:
fixed in https://github.com/blue-oil/blueoil/pull/225/commits/d3526e900ba746e8c678b91ac4a6fee927152f2b